easybuild/tools/filetools.py

```diff
@@ -49,7 +49,14 @@
 import sys
 import tempfile
 import time
-import urllib2
+try:
+    import requests
+    from requests.exceptions import HTTPError
+    HAVE_REQUESTS = True
+except ImportError:
+    import urllib2
```

Comment:
Does it make sense to add another catch here, to give more useful feedback? Something like "Can't import requests nor urllib2. If you are in an old system please install python-requests"

Reply:
`urllib2` is always available (standard library). But perhaps the urlopen exception handler can check for something like `<urlopen error [Errno 1] _ssl.c:492: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>` and if so, report
"SSL issues with urllib2. If you are using RHEL/CentOS 6.x please install the `python-requests` and `pyOpenSSL` RPM packages" and try again?